### DISCUSSION TITLE
Multiple vehicles in 1 go

### DISCUSSION THREAD

**Discussion by humuhumunukunuku** [voxeltycoon/issues#315] [General] [answered]
As a business tycoon I’d like to be able to open up new route and add multiple vehicles to this route in a series of steps as this will allow me to quickly generate more cash and save me a bunch of time and rsi as I don’t have to repeat the same steps for several times.

For example:
1. Create freight station at coal plant.
2. Create freight station at coal mine.
3. Create vehicle depot near coal mine.
4. As I’m buying a specific type of vehicle best suited to carry coal I can set the number to buy.
5. In the route screen i can assign all newly purchased vehicles to a route, or vice versa, i can assign a route to multiple vehicles.
6. Hit go for all vehicles so cash will flood in.

Thank you

**Comment by katepolak** [comment, marked as answer]
You can already easily buy vehicles in bulk by cloning them, which in my opinion is a rather intuitive way to do this, better then buying multiple vehicles, especially as that would mean buying multiple unconfigured vehicles and then adding them to a route adding an extra step compared to the cloning method.

I will close this issue now because as far as I can tell this is already in the game, but if you disagree feel free to comment bellow and we can reopen it.

**Comment by humuhumunukunuku** [comment]
@polak-jan my bad you are absolutely right. Should have hovered all the icons. Thanks!
